client/js/keybinds.js

```diff
@@ -166,7 +166,7 @@ const colorsHotkeys = {
 };
 
 for (const hotkey in colorsHotkeys) {
```

Comment:
I know this is not your PR, but I don't think this should be named `colorsHotkeys` anymore...